assets/scss/header.scss

```diff
@@ -276,6 +276,10 @@
     //make this a min-width to account for the longer header texts than "GOV.UK"
     width: unset;
     min-width: 33.33%;
+    @include govuk-media-query($from: desktop) {
+      // space for search box
+      max-width: calc(100% - 285px);
+    }
 
```

Comment:
Does this account for when someone doesn't add a search box in the header?

Reply:
No, the header will be restricted as if there is a search box.  I'll have a think about this.  Thanks.

Reply:
I have changed the CSS so this restriction is only when search boxes are shewn.